feat: Add tooltip with icon for embedding scenes

Kartik1397 opened the pull request:
Fixes #2232 
## Screenshot

![Screenshot from 2020-12-13 21-38-52](https://user-images.githubusercontent.com/24036721/102017334-5f636f80-3d8c-11eb-90b0-dd8891aa8df8.png)
![Screenshot from 2020-12-13 21-38-55](https://user-images.githubusercontent.com/24036721/102017336-612d3300-3d8c-11eb-9398-f967f6c1bce7.png)
![Screenshot from 2020-12-13 21-39-01](https://user-images.githubusercontent.com/24036721/102017337-61c5c980-3d8c-11eb-9a72-f78e3196cc3d.png)

dwelle commented:
Looks nice. I'd increase the padding from whatever it was previous.

Btw, is anyone seeing this weird glitch, where the checkboxes jump a bit on hover? Only happens in this PR and when the console is not open (Chrome stable):

![export_dialog_options_jump](https://user-images.githubusercontent.com/5153846/102017972-7d1fdd00-3d6a-11eb-9f82-c4ab75be7bac.gif)

Kartik1397 commented:
I have changed padding from 5px to 8px and also fixed bug caused by faulty scss for handle draging.
why this changelog checks are failing?



> Btw, is anyone seeing this weird glitch, where the checkboxes jump a bit on hover? Only happens in this PR and when the console is not open (Chrome stable):
>
![Peek 2020-12-13 23-21](https://user-images.githubusercontent.com/24036721/102019534-10bcd200-3d9a-11eb-8953-0478a7ba7352.gif)
This also happens on https://excalidraw.com/ but in my case checkbox are not coming down to initial position.

lipis commented:
not visible on mobile?

Kartik1397 commented:
> not visible on mobile?

https://github.com/excalidraw/excalidraw/issues/2232#issuecomment-744021951

lipis commented on src/components/ToolIcon.scss:
```diff
@@ -181,6 +183,17 @@
+    height: 0.9em;
+    margin-left: 5px;
+    margin-top: 1px;
+
```
Maybe `cursor: help;`

Kartik1397 replied:
Nice suggestion.

Kartik1397 replied:
It would look like this
![Peek 2020-12-14 03-27](https://user-images.githubusercontent.com/24036721/102025277-d7498e00-3dbc-11eb-8471-3de197a03c27.gif)

lipis replied:
Doesn't look good.. maybe skip it :)

lipis commented:
What do you guys think of showing this tooltip in other places as well. Like here: 

<img width="184" alt="Screenshot 2020-12-13 at 9 58 44 PM" src="https://user-images.githubusercontent.com/125676/102022274-625f5f80-3d8e-11eb-8023-d810c43fc556.png">

It is much faster than the `title` attribute. (Something for another PR maybe)

dwelle commented:
For buttons it shouldn't be immediate as it'd be distracting. ~200ms delay or similar is usually used.

dwelle commented:
thanks @Kartik1397 :heart:
